ts/components/ui/ListItemTransaction.tsx

```diff
@@ -71,6 +71,24 @@ type LeftComponentProps = {
   logoIcon: PaymentLogoIcon;
 };
 
+const getBadgeTextFromStatus = (transactionStatus: string) => {
```

Comment:
It's better to use the status type in the argument, just to avoid any possible issue, imho

```suggestion
const getBadgeTextFromStatus = (transactionStatus: ListItemTransactionStatus) => {
```